src/components/Editor/Timezonepicker.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+	<div>
+		<div v-if="show">
+			<Multiselect :options="timezones" />
+			<Multiselect :options="timezones" />
+		</div>
+		<button v-tooltip="t('calendar', 'set timezone')" class="icon-timezone" @click="show = !show" />
+	</div>
+</template>
+
+<script>
+import { Multiselect } from 'nextcloud-vue'
+export default {
+	name: 'Timezonepicker',
```

Comment:
We already have a Timezone selection component: https://github.com/nextcloud/calendar/blob/vue/src/components/Shared/TimezoneSelect.vue